src/components/tables/examples/TableExamples.tsx

```diff
@@ -7,7 +7,7 @@ import * as _ from 'underscore';
 import { ITableData, ITableState, ITablesState, ITableCompositeState } from '../TableReducers';
```

Comment:
Can you add an example with a disabled row?

Reply:
I have added an example